Formula/c/consul.rb

```diff
@@ -23,7 +23,7 @@ class Consul < Formula
   end
 
   # https://www.hashicorp.com/blog/hashicorp-adopts-business-source-license
-  deprecate! date: "2024-02-15", because: "will change its license to BUSL on the next release"
+  disable! date: "2024-02-15", because: "will change its license to BUSL on the next release"
```

Comment:
Guessing it's this now
```suggestion
  disable! date: "2024-02-15", because: "changed its license to BUSL"
```